cmd/sd/internal/api/client.go

```
@@ -0,0 +1,176 @@
+	endpoint, _, err := parseEndpoint(rawEndpoint)
+	return endpoint, err
+}
+
+func parseEndpoint(raw string) (string, string, error) {
+	parsed, err := url.Parse(strings.TrimSpace(raw))
+	if err != nil {
+		return "", "", err
+	}
+
+	if parsed.Scheme != "http" && parsed.Scheme != "https" {
+		return "", "", fmt.Errorf("Storyden API URL must use http or https")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)